doc/en/components/grids/_shared/advanced-filtering.md

```diff
@@ -469,10 +494,18 @@ In case you would like to change some of the colors, you need to set a class for
 <igc-grid class="grid"></igc-grid>
 ```
 
+```html
+<igc-tree-grid class="grid"></igc-tree-grid>
+```
+
 ```razor
 <IgbGrid class="grid"></IgbGrid>
 ```
 
+```razor
```

Comment:
I believe this snippet should be restricted to Tree Grid only, as it appears in the datagrid doc as well.